README.md

```diff
@@ -324,6 +326,23 @@ curl -v -H "Authorization: token=$(dcos config show core.dcos_acs_token)" -X POS
 
 **Note:** The upgrade process will cause all of your Cassandra node processes to restart.
 
+If you are upgrading to or beyond 1.0.13-X.Y.Z of DC/OS Cassandra from an older version (pre `1.0.13-X.Y.Z`), here are three possible upgrade paths:
+
+## 1. Backup/Restore assisted
+
+1. Perform [Backup Operation](https://github.com/mesosphere/dcos-cassandra-service#backup) on your currently running Cassandra Service. Please make a note of the backup name and backup location.
+2. [Install a new Cassandra Service](https://github.com/mesosphere/dcos-cassandra-service#multiple-cassandra-cluster-installation) instance.
+3. Perform [Restore operation](https://github.com/mesosphere/dcos-cassandra-service#restore) on the new cluster created in Step #2
+4. Once the restore operation is finished, please check if the data is restored correctly.
+5. Optionally, [uninstall](https://github.com/mesosphere/dcos-cassandra-service#uninstall) old cluster.
+
+## 2. Multi-DC Replication assisted
+
+1. [Install a new Cassandra Service in Multi-DataCenter configuration](https://github.com/mesosphere/dcos-cassandra-service#multi-datacenter-deployments), so that if becomes a peer DataCenter of your original Cassandra Service
+2. Cassandra Service created is step #1 will start replicating data from your original cluster, once it's online. 
+3. Once data replication finishes, perform data validation.
+4. Optionally, [uninstall](https://github.com/mesosphere/dcos-cassandra-service#uninstall) old cluster.
```

Comment:
Is this really optional?  Seems recommended; recommend dropping the optional word.